corehq/apps/app_manager/management/commands/find_broken_builds.py

```diff
@@ -44,6 +44,10 @@ def form_filter_error(build):
             prev_form_filter = form_filter
 
 
+def all_builds(build):
```

Comment:
Perhaps this function could be named something more descriptive regarding what it's for? Alternatively a docstring would be helpful.

```suggestion
def fail_all_builds(build):
```